conanfile.py

```diff
@@ -98,6 +98,13 @@ def _setup_args_gn(self):
         self._set_env_arg(args, "CXXFLAGS", "extra_cflags_cc")
         self._set_env_arg(args, "CXXFLAGS", "extra_cflags_objcc")
         self._set_env_arg(args, "LDFLAGS",  "extra_ldflags")
+        
+        args += [ "custom_conan_compiler_args_file=\\\"%s\\\"" % "@conanbuildinfo.args" ]
```

Comment:
```suggestion
        args += [ "custom_conan_compiler_args_file=\\\"@%s\\\"" % os.path.join(self.install_folder, "conanbuildinfo.args") ]
```
With an absolute path, the file should always be found